elasticdl/python/worker/main.py

```diff
@@ -24,12 +25,27 @@ def main():
             channel = build_channel(addr)
 
             # Wait the channel is ready by a Future object.
-            grpc.channel_ready_future(channel).result()
-            logger.info(
-                "grpc channel %s to connect pod %s is ready"
-                % (addr, addr.split(".")[0])
-            )
-            ps_channels.append(channel)
+            connect_succeed = False
+            for i in range(3):
```

Comment:
Move this 3 and timeout=60 to constants in this file?

Reply:
Yeah